### ISSUE TITLE
CORS error for downloading attachments on MediaLibrary from AWS S3 bucket

### ISSUE THREAD

**Issue by vinod77** [strapi/strapi#10042]
## Bug report
CORS error for downloading attachments on MediaLibrary from AWS S3 bucket.

### Describe the bug

We migrated our Strapi app to AWS, we are not able to download attachments from Media Library, from AWS S3 bucket. 
It's throwing CORS error.

### Error:
Access to XMLHttpRequest at '<S3_bucket>.<S3_region>.amazonaws.com/<file_name>.jpeg' from origin '<our_domain>' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

### Steps to reproduce the behavior

1. Run Strapi app connecting to AWS S3 Bucket and Mongo DB.
2. Open Media Library and try to download any attachment
3. Throws CORS error 

### Expected behavior

Attachments from Media Library, S3 bucket can be downloaded.

### System

- Node.js version: v14.8.0
- NPM version: 6.14.7
- Strapi version: v3.0.6
- Database: Mongo
- Operating system: MAC

**Comment by derrickmehaffy** [comment]
I'm going to set this as a duplicate of https://github.com/strapi/strapi/issues/10044
